docs/operations/cli.rst

```diff
@@ -25,6 +25,10 @@ following are the command line options that Envoy supports.
 
   *(optional)* The output file path where the admin address and port will be written.
 
+.. option:: --local-address-ip-version <string>
+
+  *(optional)* The local address IP version. The options are ``v4`` or ``v6``. Defaults to ``v4``.
```

Comment:
Can you add some documentation on how this is used by Envoy? I.e. which headers it affects etc.